src/op/nms.h

```diff
@@ -33,27 +32,30 @@ class nms
                 ADDR_IPV6,
                 ADDR_NONE
         } addr_t;
+        struct cmp_in6_addr
+        {
+                bool operator()(const in6_addr& a,
+                                const in6_addr& b) const
+                {
+                        return (0 > memcmp(&a.s6_addr,
+                                           &b.s6_addr,
+                                           sizeof(a.s6_addr)));
+                }
+        };
+        typedef std::set<in_addr_t> ipv4_set_t;
```

Comment:
prefer moving it to private like it used to be, if there isn't any specific reason..